pkg/cloudprovider/aws/instanceprofile.go

```diff
@@ -28,6 +26,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
```

Comment:
This is in the wrong place. It should be up by other golang native imports.

Reply:
I agree. I just ran `make ci` and it moved the strings down there, so that it could sort it alphabetically. In other words, to separate the imports, it's important to have a newline before running, or it won't sort it properly.